tests/functional_test.py

```diff
@@ -3,6 +3,7 @@
   The adaguc-server executable is ran with various settings and configuration files.
   Results are checked against expected results and should not differ.
 """
+
 import unittest
 import sys
 from AdagucTests.TestWMSTimeSeries import TestWMSTimeSeries
```

Comment:
- Can you add a functional test demonstrating this dataset?
- Can you add documentation to https://github.com/KNMI/adaguc-server/tree/master/doc/format_standard ?